docs/src/pages/[platform]/theming/style-props/examples/PositionExamples.tsx

```diff
@@ -0,0 +1,43 @@
+import { Button, View, useTheme } from '@aws-amplify/ui-react';
+
+// use any CSS value
+export const PositionStylePropExample = () => {
+  return (
+    <View position="relative" width="100%" height="4.5rem">
```

Comment:
I think this needs a border around the parent div so it's clear what the absolute positioning is working against:
![image](https://user-images.githubusercontent.com/6165315/174460988-d9ad0528-a4d5-4280-94d1-2b975e9f028e.png)
```suggestion
    <View
      border="1px solid #000"
      position="relative"
      width="100%"
      height="5rem"
    >
```

Reply:
border added